test/integration/targets/setup_zabbix/templates/zabbix_server.conf.j2

```diff
@@ -0,0 +1,650 @@
+# This is a configuration file for Zabbix server daemon
```

Comment:
Nearly all of this file is commented out, can we trim it? Rather than including a 650 line file that is mostly not used?

Reply:
I used the default config file with the thinking that it could be easier to diff against new versions of Zabbix once they are released. It is probably not worth it though, therefore I just trimmed it down to the 5 actually used lines.